app/views/items/index.html.erb

```diff
@@ -2,6 +2,6 @@
 
 <% @items.each do |item| %>
   <ul>
-    <li> <%= item.name %> </li>
+    <li> <%= link_to item.name, "/merchants/#{@merchant.id}/items/#{item.id}" %> </li>
```

Comment:
If you're not comfortable with path helpers then I would keep it this way but if you are I might change the path on this link.